tests/constants_t.py

```diff
@@ -0,0 +1,20 @@
+#-*- coding: utf-8 -*-
+
+import unittest
+
+from loris import constants
+
+class TestImageRequest(unittest.TestCase):
+
+    def test_valid_filenames(self):
+        self._assert_valid('/123.jpg/full/full/0/default.jpg')
+
+    def test_valid_special_characters(self):
```

Comment:
Also in the pytest front, it might be nice to use `@pytest.mark.parametrized` here, so that each of these case runs as an isolated test.